ReflectionAnalyzers/ReflectionAnalyzers.csproj

```diff
@@ -8,10 +8,14 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <BuildOutputTargetFolder>analyzers\dotnet\cs</BuildOutputTargetFolder>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageOutputPath>$(SolutionDir)</PackageOutputPath>
```

Comment:
👎 Leave it in the output directory.

Reply:
Yes, simplest. No reason to go 1:1 with what was before.

Reply:
Thanks for catching. This was supposed to be one of my comments last night to recommend removing this. I'll fix it in the same commit as all the other items that move away from the prior behavior.

Reply:
@JohanLarsson Do you still want GeneratePackageOnBuild in release mode? My own preference would be to write `dotnet pack ReflectionAnalyzers -c Release` (or script it, rather).

Reply:
No, remove it, it only slows down build & test for no benefit. Dunno why I added it, probably a copy paste accident.